src/main/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjection.java

```diff
@@ -35,7 +36,7 @@
     private static final Pattern IP_ADDRESS_PATTERN =
             Pattern.compile("\\b((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\\.|$)){4}\\b");
 
-    private StringBuilder getResponseFromPingCommand(String ipAddress, Supplier<Boolean> predicate)
+    public StringBuilder getResponseFromPingCommand(String ipAddress, Supplier<Boolean> predicate)
```

Comment:
can be made package private.

Reply:
sure, i will change to private static final later, having dinner now . ha

Reply:
oh , i change to public here because i need to mock this method, in order to verify this method is called with Supplier (false or true ) , otherwise there is no way to verify getResponseFromPingCommand is called with expected predicate value.

Reply:
Got that point but i think making package-private/Default can help achieve the same and also restrict access outside the package.

Reply:
default works